Add project-xeric to plugin-hub

Septem151 opened the pull request:
Project Xeric is a plugin designed for the "**Zeah Ironmen**" clan. The plugin features task tracking, a ranking system, leaderboards, and account exception tracking.

The plugin adds a sidebar panel that features two tabs: a Player Profile section which includes a task list of each different tier of task and a player status card, and a Leaderboard section that has a link to the WiseOldMan group for the clan.

Picture of the Player Profile section:
![example](https://github.com/user-attachments/assets/98ced38b-0c8d-4b5e-90be-053a23dbdf30)

Picture of the Leaderboard section:
<img width="276" height="244" alt="image" src="https://github.com/user-attachments/assets/e78e6f12-51b8-468c-97d9-87918c61a5cf" />

Picture of the plugin's Configuration settings:
<img width="277" height="219" alt="image" src="https://github.com/user-attachments/assets/08fe6b6b-a3a6-4db6-80e6-ff50dc62e9b8" />

Picture of chat messages upon completing tasks:
<img width="667" height="178" alt="image" src="https://github.com/user-attachments/assets/f3e90fa9-39d8-4820-bf2e-e42ae988bbe5" />
<img width="496" height="63" alt="image" src="https://github.com/user-attachments/assets/31bed47e-2ae7-46ef-a329-3fb204d0bec8" />
<img width="634" height="170" alt="image" src="https://github.com/user-attachments/assets/027a3a0c-717a-429e-b9f5-0bd37d84803b" />

LlemonDuck commented:
https://github.com/Septem151/project-xeric/blob/2332fd8f1a2f76847ec4409a90f52cf2b9415fdd/src/main/java/io/septem150/xeric/data/ProjectXericApiClient.java#L42

LlemonDuck commented on plugins/project-xeric:
```diff
@@ -0,0 +1,2 @@
+repository=https://github.com/Septem151/project-xeric.git
+commit=2332fd8f1a2f76847ec4409a90f52cf2b9415fdd
```
```suggestion
commit=2332fd8f1a2f76847ec4409a90f52cf2b9415fdd
warning=This plugin submits your IP address and player data including username to a server not controlled or verified by the RuneLite developers.
```

LlemonDuck commented:
Looks like you're transitively registering SummaryPanel to the event bus but not unregistering it in shutDown, which you need to ensure as well. It's generally easier to keep track of these things if you can place them all in your plugin class instead of a bunch of subcalls.

Septem151 commented:
Done, instead of registering the panel(s) within their own class, I moved the eventBus registration/de-registration to the plugin class like you said. I also removed a bunch of unused code that isn't ready yet to make the review a little easier.
